recipes/LibriSpeech/SSL/hparams/train_wav2vec.yaml

```
@@ -0,0 +1,95 @@
+seed: 1986
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+data_folder: /idiap/resource/database/LibriSpeech
+save_folder: data
+output_folder: exp-test
+
+train_splits: ["train-clean-100", "train-clean-360", "train-other-500"]
+dev_splits: ["dev-clean"]
+test_splits: ["test-clean", "test-other"]
+train_csv: !ref <output_folder>/train.csv
+valid_csv: !ref <output_folder>/dev-clean.csv
+test_csv:
+   - !ref <output_folder>/test-clean.csv
+   - !ref <output_folder>/test-other.csv
+skip_prep: true
```

```suggestion
skip_prep: False
```

Otherwise won't work plug and play.

---

good catch, done